client/lib/abtest/active-tests.js

```diff
@@ -108,4 +108,14 @@ export default {
 		},
 		defaultVariation: 'disabled',
 	},
+	domainSuggestionKrakenV321: {
+		datestamp: '20180524',
+		variations: {
+			domainsbot: 1,
```

Comment:
Would it be better to set this as `domainsbot: 0`?

Reply:
You are absolutely right! I'll fix that and deploy :)